### PR TITLE
Analyze sex and goodbad

### PR DESCRIPTION
**Pull request by markus-hinsche**
**Motivation:** Besides model evaluation on age (https://github.com/Welthungerhilfe/cgm-ml/pull/186), we can now easily evaluate on:
- sex ('male' or 'female')
- good/bad scan (quality of the scan)

**Changes:**
- enable sex analysis
- enable good/bad analysis

**Findings**
(As expected) The accuracy is greater on good scans. Therefore we should strive to get good scans:

There is little or no difference between female and male scans:

**Results:**
- https://dev.azure.com/cgmorg/ChildGrowthMonitor/_build/results?buildId=2166&view=artifacts&pathAsName=false&type=publishedArtifacts

**Do before merge:** Revert `pr: none`

### CODE REVIEW

File: src/common/evaluation/QA/eval_depthmap_models/src/utils.py [hunk 1]
```diff
@@ -124,6 +143,48 @@ def calculate_and_save_results(MAE: pd.DataFrame, complete_name: str, CSV_OUT_FP
     result.to_csv(CSV_OUT_FPATH, index=True)
 
 
+def calculate_performance_gender(code: str, df_mae: pd.DataFrame, RESULT_CONFIG: Bunch) -> pd.DataFrame:
+    df_mae_filtered = df_mae.iloc[df_mae.index.get_level_values('scantype') == code]
+    accuracy_list = []
+    accuracy_thresh = 1.0
```
Comment (gangooteli) [inline src/common/evaluation/QA/eval_depthmap_models/src/utils.py:149]: What is the significance of the accuracy_thresh?
Is there can be diff value of accuracy_thresh other than 1.0 ?

Comment (markus-hinsche) [reply src/common/evaluation/QA/eval_depthmap_models/src/utils.py:149]: Yes 1.0 means 1cm (centimeter). Errors below 1cm count as "accurate enough"

Comment (gangooteli) [reply src/common/evaluation/QA/eval_depthmap_models/src/utils.py:149]: Will it be appropriate to add this in config file?

Comment (markus-hinsche) [reply src/common/evaluation/QA/eval_depthmap_models/src/utils.py:149]: will do, any other comment?

Comment (gangooteli) [reply src/common/evaluation/QA/eval_depthmap_models/src/utils.py:149]: Sounds good !!
No further suggestions

**Review by gangooteli — APPROVED** [review]
LGTM. Feel free to make suggested changes.

**Pull request merged**